deploy/managed-upgrade-operator-config/10-managed-upgrade-operator-configmap.yaml

```diff
@@ -28,31 +28,53 @@ data:
       expectedNodeDrainTime: 8
     healthCheck:
       ignoredCriticals:
-      - DNSErrors05MinSRE
       - MetricsClientSendFailingSRE
-      - UpgradeNodeScalingFailedSRE
       - PruningCronjobErrorSRE
-      - PrometheusRuleFailures
-      - CannotRetrieveUpdates
-      - FluentdNodeDown
-      # OSD-11927
-      - ClusterProxyCAExpiringSRE
-      # https://bugzilla.redhat.com/show_bug.cgi?id=1986981
-      - PrometheusRemoteWriteBehind
-      - KubePersistentVolumeErrors
-      - PrometheusBadConfig
-      - PrometheusRemoteStorageFailures
-      - AlertmanagerMembersInconsistent
-      - AlertmanagerClusterFailedToSendAlerts
-      - AlertmanagerConfigInconsistent
-      - AlertmanagerClusterDown
-      - KubeStateMetricsListErrors
-      - KubeStateMetricsWatchErrors
-      - ThanosRuleSenderIsFailingAlerts
-      - ThanosRuleHighRuleEvaluationFailures
-      - ThanosNoRuleEvaluations
-      # OSD-13649
       - etcdGRPCRequestsSlow
+      # OSD-22494
+      - AlertmanagerFailedReload
+      - CloudIngressOperatorOfflineSRE
+      - ClusterMonitoringErrorBudgetBurnSRE
+      - ConfigureAlertmanagerOperatorOfflineSRE
+      - console-ErrorBudgetBurn
+      - ControlPlaneNodeFileDescriptorLimitSRE
+      - ControlPlaneNodeFilesystemAlmostOutOfFiles
+      - ControlPlaneNodeFilesystemSpaceFillingUp
+      - CustomerWorkloadPreventingDrainSRE
+      - EbsVolumeBurstBalanceLT20PctSRE
+      - EbsVolumeStuckAttaching10MinSRE
+      - EbsVolumeStuckDetaching10MinSRE
+      - etcdHighNumberOfFailedGRPCRequests
+      - ExcessiveContainerMemoryCriticalSRE
+      - ExtremelyHighIndividualControlPlaneCPU
+      - ExtremelyHighIndividualControlPlaneMemory
+      - HAProxyDownSRE
```

Comment:
HAProxyDownSRE should be a blocker for the upgrade

Reply:
Remove the alert from the list